src/main/kotlin/com/plus/taxiapp/infra/store/member/MemberEntity.kt

```diff
@@ -0,0 +1,20 @@
+package com.plus.taxiapp.infra.store.member
+
+import com.plus.taxiapp.domain.Account
+import com.plus.taxiapp.domain.PaymentType
+import com.plus.taxiapp.infra.store.payment.AccountEntity
+import jakarta.persistence.*
+
+@Entity
+@Table(name = "member")
+class MemberEntity(
+    @Id @GeneratedValue(strategy = GenerationType.IDENTITY)
+    var id: Long? = null,
+    var name: String,
+) {
+    @Enumerated(EnumType.STRING)
+    var defaultPaymentType: PaymentType? = null
+    var defaultPaymentId: Long? = null
+    @OneToMany(mappedBy = "memberEntity")
```

Comment:
🚫 **[ktlint]** <standard:spacing-between-declarations-with-annotations> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Declarations and declarations with annotations should have an empty space between.